cmd/kubeadm/app/componentconfigs/defaults.go

```diff
@@ -33,6 +34,25 @@ import (
 const (
 	// KubeproxyKubeConfigFileName defines the file name for the kube-proxy's kubeconfig file
 	KubeproxyKubeConfigFileName = "/var/lib/kube-proxy/kubeconfig.conf"
+
+	// KubeletReadOnlyPort specifies the default insecure http server port
+	// 0 will disable insecure http server.
+	KubeletReadOnlyPort int32 = 0
+
+	// KubeletRotateCertificates specifies the default value to enable certificate rotation
+	KubeletRotateCertificates = true
+
+	// KubeletAuthenticationAnonymousEnabled specifies the default value to disable anonymous access
+	KubeletAuthenticationAnonymousEnabled = false
+
+	// KubeletAuthorizationMode specifies the default authorization mode
+	KubeletAuthorizationMode = kubeletconfigv1beta1.KubeletAuthorizationModeWebhook
+
+	// KubeletAuthenticationWebhookEnabled set the default value to enable authentication webhook
+	KubeletAuthenticationWebhookEnabled = true
+
+	// KubeletHealthzBindAddress specifies the default healthz bind address
+	KubeletHealthzBindAddress = "127.0.0.1"
 )
 
 // DefaultKubeProxyConfiguration assigns default values for the kube-proxy ComponentConfig
```

Comment:
Can you add the warning elses for `DefaultKubeProxyConfiguration` too?

Reply:
Thanks a lot, fixed.